mne_bids/meg_bids.py

```diff
@@ -305,25 +318,14 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
                                      verbose=verbose)
 
     # create filenames
-    scans_fname = make_bids_filename(
-        subject=subject_id, session=session_id, suffix='scans.tsv',
-        prefix=ses_path)
-
-    coordsystem_fname = make_bids_filename(
```

Comment:
I'm not a big fan of using an object here, but maybe it's just me. @agramfort any opinions?